Make prerequisites clearer (#585)
ros-planning/moveit2_tutorials: doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

Getting Started
---------------

> If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>` or our :doc:`Docker Guide </doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu>`. If you followed the Docker Guide, also follow the second part of the Getting Started tutorial (from :doc:`Create a Colcon Workspace and Download Tutorials <https://moveit.picknik.ai/humble/doc/tutorials/getting_started/getting_started.html#create-a-colcon-workspace-and-download-tutorials>` onwards) to set up the tutorials. As of Sep 26, 2022, ensure you have enabled Cyclone DDS as described there.

Reviewer: ```suggestion
If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>` or our :doc:`Docker Guide </doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu>`. If you followed the Docker Guide, also follow the second part of the Getting Started tutorial (from :ref:`Create a Colcon Workspace and Download Tutorials <doc/tutorials/getting_started/getting_started.html:create a colcon workspace and download tutorials>` onwards) to set up the tutorials. As of Sep 26, 2022, ensure you have enabled Cyclone DDS as described there.
```
This lets Sphinx handle the reference between documents
